packages/theme-patternfly-org/pages/global-css-variables.md

```diff
@@ -52,6 +83,22 @@ Breakpoint suffixes are used in utility classes and layouts as well as in many c
 
 PatternFly 4 styles provide a default starting point. You can use the variable system to make adjustments to that default styling. When you change one or more elements, you should package those values into a new SCSS stylesheet to replace the default styling.
 
+Overrides to PatternFly variables should be made at the `:root` level (for global variables) or at the top-level component selector (for component variables, ex: `.pf-c-button`), as these overrides will cascade down to children elements accordingly.
+
+Example:
```

Comment:
This is singular. I think it could be either, even though there's only one example block, there are multiple lines which could be examples. Either way it should probably be consistent.